src/views/AddLiquidity/index.tsx

```diff
@@ -73,6 +73,7 @@ export default function AddLiquidity() {
   const tokens = serializeTokens()
 
   const [zapMode] = useZapModeManager()
+  const [temporarilyZapMode, setTemporarilyZapMode] = useState(zapMode)
```

Comment:
please avoid a useState default value is another state.
how about this
`const [temporarilyDisableZapMode, setTemporarilyDisableZapMode] = useState(false)`

to know if zapMode is on
check `!!zapModa || !temporarilyDisableZapMode`